crypto/verificationhelper/verificationhelper.go

```diff
@@ -141,6 +141,70 @@ func (vh *VerificationHelper) getLog(ctx context.Context) *zerolog.Logger {
 	return &logger
 }
 
+func (vh *VerificationHelper) checkTransactionID(ctx context.Context, evt *event.Event) *VerificationTransaction {
+	log := vh.getLog(ctx).With().
+		Str("verification_action", "check transaction ID").
+		Stringer("sender", evt.Sender).
+		Stringer("room_id", evt.RoomID).
+		Stringer("event_id", evt.ID).
+		Stringer("event_type", evt.Type).
```

Comment:
Just a note, this change looses the `event_type` key from the callback's logger. I'm not sure how useful it is (and it might be added to the context somewhere else).